.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,24 @@
+{
+    "configurations": [
+        {
+            "name": "switch",
+            "includePath": [
+                "${default}",
+                "${workspaceFolder}/**",
+                "${DEVKITPRO}/libnx/include/",
+                "${DEVKITPRO}/portlibs/switch/include/",
+                "${workspaceFolder}/sys-tune/nxExt/include/",
+                "${workspaceFolder}/sys-tune/source/",
+                "${workspaceFolder}/overlay/lib/include/",
+                "${workspaceFolder}/overlay/source/",
```

Comment:
I don't think a source folder should be in this.

Reply:
Good catch